src/components/OmniSearch/__tests__/OmniSearch.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import OmniSearch from "../"
+import { render } from "../../../common/TestUtil"
+import { cleanup, fireEvent } from "@testing-library/react"
+
+const baseProps = {
+  visible: true,
+  value: "demo",
+  onChange: () => null,
+  onClose: () => null,
+}
+
+describe("OmniSearch", () => {
+  afterEach(cleanup)
+
+  it("renders correctly when visible is false", () => {
+    const { baseElement } = render(
+      <OmniSearch {...baseProps} visible={false} />
+    )
+    expect(baseElement).toMatchSnapshot()
+  })
+
+  it("renders correctly when visible is true", () => {
+    const { baseElement } = render(<OmniSearch {...baseProps} />)
```

Comment:
Can we get rid of most of the other snapshots? they seem to almost same.

Reply:
Removed 2/5 snapshots. Only three that remain are - not visible, visible with no dropdown, visible with dropdown.